test/integration/emulator/FakeRecordingEndpoint.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+#include "transport/RecordingEndpoint.h"
+
+class FakeRecordingEndpoint : public transport::RecordingEndpoint
+{
+public:
+    FakeRecordingEndpoint() : _name("FakeRecordingEndpoint") {}
```

Comment:
If we expand this a bit we could actually setup a recording sink in the fake network and verify that the data arrives as expected. We also get much better code coverage of the API and mixer setup.